cluster_migration_core/cluster_migration_core/robot_actions/cluster_action_executor.py

```diff
@@ -0,0 +1,25 @@
+from robot import run
+
+from cluster_migration_core.robot_actions.action_executor import ActionExecutor
+
+
+class ClusterActionExecutor(ActionExecutor):
+    """
+    Use this class to execute operations against a cluster
+    """
+
+    def __init__(self, hostname: str, port: int, *args, **kwargs):
+        self.hostname = hostname
+        self.port = port
+        super().__init__(*args, **kwargs)
+
+    def _execute(self) -> None:
+        run(
+            self.actions_dir,
+            include=self.include_tags,
+            exclude=self.exclude_tags,
+            outputdir=self.output_dir,
+            variable=[f"host:{self.hostname}", f"port:{self.port}"],
+            consolewidth=self.consolewidth,
+            loglevel=self.loglevel
```

Comment:
My understanding is that this file replaces the old "test_executor.py"? I like this add.

Reply:
Yep, that's the plan!